provider/inspec.rb

```diff
@@ -21,6 +21,7 @@
 module Provider
   # Code generator for Example Cookbooks that manage Google Cloud Platform
   # resources.
+  # rubocop:disable Metrics/ClassLength
```

Comment:
We should start having the Magician run Rubocop

Reply:
Next PR includes CI changes for running unit tests, can add it there